HelpAssistHer/help-assist-her
# Add placeholder for phone number field

## Description
The phone number needs to be in a specific format to pass validation. This PR just adds a placeholder to the phone number field so the user knows what that format is. We can make this more user friendly in the future, but this is the easiest way right now to make sure there aren't format related errors for the user.

## Test Plan
1. Go to `http://localhost:4000/verification`
1. Verify there is placeholder text in both Phone Number fields indicating the format the phone number needs to be in.
